verilog/tools/ls/verilog-language-server_test.cc

```diff
@@ -33,12 +36,27 @@ namespace {
 
 using namespace nlohmann;
 
+// FIXME (glatosinski) use more plausible modules
+constexpr absl::string_view  //
```

Comment:
Similar to the comment in the symbol table handler test: make the variable separate. Also, use raw string literals.